.travis.yml

```diff
@@ -3,16 +3,16 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
   - 1.1
   - 1.2
+  - 1.3
```

Comment:
How about changing this to `1` so that it always tracks the latest stable release?

Reply:
Done!